pkg/arvo/lib/hood/drum.hoon

```diff
@@ -223,7 +224,7 @@
   ==
 ::
 ++  on-load
-  |=  ver=?(%1 %2)
+  |=  ver=?(%1 %2 %3)
```

Comment:
This `+on-load` is... quite something. At the very least you'll need to update `/app/hood` to actually call this.

Should probably, for `%3`, only start the s3-store? Though I guess we can't guarantee that the others didn't get skipped over, because of how OTA fast-forward right now...

Reply:
The usual pattern is a series of

```
=?  state  =(ver %1)
  ...(ver %2)
=?  state  =(ver %2)
  ...
```

People do in practice skip large numbers of commits.  On mainnet right now the only update that has to be staged is the compiler upgrade in january.  You can take the original pill, jump to that commit, then jump to the present, and it'll just work.

Reply:
Can someone verify that I updated hood properly? I think I did and it works well, but it's always nice to have another pair of eyes.

Reply:
I don't think so.  I think you need to add another case to +on-load in app/hood or else this won't get called.  There's no magic that calls +on-load in drum right now, we just manually call it as necessary.  Also the overall state in app/hood is what needs its version number bumped.  Basically, the flow is:

- Bump app/hood's whole state's version number in +on-save and +hood-old
- In app/hood's +on-load, use this version number change as a trigger to do anything you need to do.  In this case, that's call +on-load in lib/hood/drum
- In lib/hood/drum, start the apps

Reply:
Thanks. I've modified it. I tested it by booting a fakezod on master, then copying in the changes and saw `%hood` successfully reload with the `%s3-store` started.